x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/support/mapper/NativeRoleMappingStore.java

```diff
@@ -296,10 +296,9 @@ private void getMappings(ActionListener<List<ExpressionRoleMapping>> listener) {
         if (securityIndex.isAvailable()) {
             loadMappings(listener);
         } else {
-            logger.info("The security index is not yet available - no role mappings can be loaded");
```

Comment:
I removed this info level logger since it possible to run a system with only file based configuration and the security index is never created. In that case (which is the default state for serverless), this message could be emitted on every authc that needs to resolve role mappings.